wordpress-mobile/gutenberg-mobile
# Force translation update when generating new bundles

While testing #1503 I noticed some translations missing. The current `yarn i18n-cache` is overly lazy when it comes to downloading translations, and it will only update them if they have never been cached before.

This might be ok during development, but we need to ensure our releases always get the right translations. IMO, this includes any betas or intermediate builds we produce.

For this, I added a `prebundle` script that runs the new `i18n-cache:force` script, and will download or update any existing translations.

To test:

- Run `yarn bundle`
- Verify that translations are always downloaded

Update release notes:
